### DISCUSSION TITLE
'Default condition should be last one' error

### DISCUSSION THREAD

**Discussion by Nithur-M** [csandman/chakra-react-select#190] [Q&A] [answered]
The package worked perfectly well but after updating to 4.2.3 it is throwing the above error. Nothing changed on my code. Why?

**Comment by csandman** [comment]
Can you give more info?  What's your environment?  Where is that error showing up, in the editor, on the cli for your app?

**Comment by Nithur-M** [comment]
the error showed up while building the project in production and also in the local environment. I am using it in Nextjs.

**Reply by csandman** [reply]
Yeah never mind, I saw the comments left on the issue, a fix will be pushed shortly!

**Comment by csandman** [comment, marked as answer]
Ok a fix for this issue was pushed out in [`v4.2.4`](https://github.com/csandman/chakra-react-select/releases/tag/v4.2.4)!  If you're still having any issues let me know.
